Automattic/wp-calypso
# BBE: Enable for WoA sites

#### Proposed Changes

* Removes the `difm/allow-woa-sites` feature flag and shows WoA sites in the BBE site picker step.
* Removes the `onboarding/difm` as a general cleanup task.

#### Testing Instructions



* Go to `/start/do-it-for-me` and select "Existing Site".
* Confirm that WoA sites are visible in the site picker.
* Confirm that self-hosted Jetpack sites are not visible in the site picker.

#### Pre-merge Checklist






Fixes Automattic/martech#1227
